charts/scorpio-broker-dist/templates/query-manager-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Values.queryManager.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    service: {{ .Values.queryManager.queryLabel }}
+  name: {{ include "query-manager.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ .Values.queryManager.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "query-manager.fullname" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  strategy:
+  {{- with .Values.queryManager.updateStrategy }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        service: {{ include "query-manager.fullname" . }}
+    spec:
+      containers:
+      - image: {{ .Values.queryManager.image.repository }}:{{ .Values.queryManager.image.tag }}
```

Comment:
Why did you decide to use that config mechanism? Its a little uncomfortable, since a user then has to override all configurations, instead of only those needed. Why not configuer the spring-boot application directly through its environment config mechanism?

Reply:
Hi, 
with the spring_args argument you would still just provide the configs that you want to change but it's than just one set config var for the user to override with his complete config instead of going through different env vars for spring boot.